src/utilities/inputParsers.ts

```diff
@@ -21,3 +27,17 @@ export const parseInputMergeMethod = (): AllowedMergeMethods => {
 
   return AllowedMergeMethods[input];
 };
+
+export const parseInputMergeCategory = (): AllowedMergeCategories => {
+  const input = getInput('MERGE_CATEGORY');
+
+  if (input.length === 0 || AllowedMergeCategories[input] === undefined) {
+    logWarning(
+      'MERGE_CATEGORY value input is ignored because its malformed, defaulting to MAJOR.',
+    );
+
+    return AllowedMergeCategories.MAJOR;
+  }
+
+  return AllowedMergeCategories[input];
```

Comment:
Do you know for sure here that input is always one of the categories?

Reply:
yeah. The github action handles that bit.